pkg/restapi/issuer/operation/operations.go

```diff
@@ -1728,12 +1736,25 @@ func (o *Operation) hanlDIDExStateMsg(msg service.StateMsg) error {
 	return nil
 }
 
-// read credential manifest from profile URL endpoints
-// TODO for now returning empty manifest, TO BE IMPLEMENTED [issue##561 & issue#563]
-func (o *Operation) readCredentialManifest() *cm.CredentialManifest {
-	return &cm.CredentialManifest{
-		ID: uuid.NewString(),
+/*
+read credential manifest issuer detail from persisted profile data and scope
+from the persisted transaction cred scope.
+*/
+// TODO issue#561 Add credential manifest presentation definition
+func (o *Operation) readCredentialManifest(profileData *issuer.ProfileData,
+	txnCredScope string) *cm.CredentialManifest {
+	for scope, cmDesciptor := range o.cmOutputDescriptor {
```

Comment:
loop can be replaced by a single line `o.cmOutputDescriptor[txnCredScope]`

Reply:
done